doc/snippets/Microsoft.Data.SqlClient/SqlAuthenticationParameters.xml

```diff
@@ -46,5 +47,9 @@
             <summary>Gets the database name.</summary>
             <value>The database name.</value>
         </DatabaseName>
+        <TimeoutInMilliseconds>
```

Comment:
Why are we using ms here? All other timeouts in this library use seconds, don't they?

Reply:
Sure, even though this is read only, makes sense to keep it in seconds.